specification/templates/CONTRIBUTING.md

```diff
@@ -72,7 +80,7 @@ any 'help wanted' issues is a great place to start.
 
 ## Licensing
 
-See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
+See the LICENSE file for our repository's licensing. We will ask you to
```

Comment:
Can you revert this change? This PR addresses it https://github.com/signalfx/gdi-specification/pull/128

Reply:
Done!